letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -58,6 +58,7 @@ for arg in "$@" ; do
     --verbose)
       VERBOSE=1;;
     -[!-]*)
+      OPTIND=1
```

Comment:
Now you're changing only the template; you should also be committing the built version. (Run build.py.)